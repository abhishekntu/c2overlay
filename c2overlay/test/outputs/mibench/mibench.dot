digraph graphname {
N10 [color="black", label="N7N16_add_N7_mul_N16_N10"];
N11 [color="black", label="O0_N17_N11"];
N8 [color="black", label="N8N14_add_N8_add_N14_N8"];
N9 [color="black", label="N6N9_add_N6_mul_N9_N9"];
N1 [color="black", label="I0_N1_N1"];
N2 [color="black", label="I1_N2_N2"];
N3 [color="black", label="I2_N3_N3"];
N4 [color="black", label="N12N13_mul_Imm_9_N12_add_Imm_1_N13_N4"];
N5 [color="black", label="N10N5_mul_Imm_2_N10_add_N5_N5"];
N6 [color="black", label="N11N4_mul_Imm_6_N11_add_Imm_43_N4_N6"];
N7 [color="black", label="mul_N15_N7"];
N10 -> N8;
N8 -> N11;
N9 -> N8;
N1 -> N10;
N1 -> N5;
N2 -> N5;
N2 -> N9;
N3 -> N4;
N3 -> N7;
N3 -> N6;
N4 -> N7;
N5 -> N9;
N6 -> N10;
N6 -> N9;
N7 -> N8;
}
