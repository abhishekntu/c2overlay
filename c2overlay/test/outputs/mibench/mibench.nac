procedure foo(in s32 x,in s32 y,in s32 z,out s32 D_2407)
{
  localvar s32 D_2406;
  localvar s32 D_2408;
  localvar s32 D_2409;
  localvar s32 D_2410;
  localvar s32 D_2411;
  localvar s32 D_2412;
  localvar s32 D_2413;
  localvar s32 D_2414;
  localvar s32 D_2415;
  localvar s32 D_2416;
  localvar s32 D_2417;
  localvar s32 temp;
L0005:
  D_2406 <= mul z,6;
  temp <= add D_2406,43;
  D_2408 <= mul y,2;
  D_2409 <= add D_2408,x;
  D_2410 <= add D_2409,temp;
  D_2411 <= mul D_2410,x;
  D_2412 <= add y,temp;
  D_2413 <= mul D_2412,y;
  D_2414 <= add D_2411,D_2413;
  D_2415 <= mul z,9;
  D_2416 <= add D_2415,1;
  D_2417 <= mul D_2416,z;
  D_2407 <= add D_2414,D_2417;
}
