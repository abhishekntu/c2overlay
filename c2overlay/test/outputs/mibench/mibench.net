.input N1
pinlist: N1
    
.input N2
pinlist: N2
    
.input N3
pinlist: N3
    
.output out:N11
pinlist: N11
    
.clb N10_blk
pinlist: N6 N1 open open N10 open open open open 
subblock: N10_blk 0 1 open open 4 open open open open

.clb N8_blk
pinlist: N7 N9 N10 open N11 open open open open 
subblock: N8_blk 0 1 2 open 4 open open open open
    
.clb N9_blk
pinlist: N5 N6 N2 open N9 open open open open 
subblock: N9_blk 0 1 2 open 4 open open open open
    
.clb N4_blk
pinlist: N3 open open open N4 open open open open 
subblock: N4_blk 0 open open open 4 open open open open

.clb N5_blk
pinlist: N1 N2 open open N5 open open open open 
subblock: N5_blk 0 1 open open 4 open open open open

.clb N6_blk
pinlist: N3 open open open N6 open open open open 
subblock: N6_blk 0 open open open 4 open open open open

.clb N7_blk
pinlist: N4 N3 open open N7 open open open open 
subblock: N7_blk 0 1 open open 4 open open open open

