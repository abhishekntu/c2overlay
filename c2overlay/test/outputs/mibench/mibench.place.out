Netlist file: test.net   Architecture file: test.arch.xml
Array size: 3 x 3 logic blocks

#block name	x	y	subblk	block number
#----------	--	--	------	------------
N1		0	1	0	#0
N2		1	0	0	#1
N3		4	2	0	#2
out:N11		4	1	0	#3
N10_blk		1	2	0	#4
N8_blk		3	1	0	#5
N9_blk		2	1	0	#6
N4_blk		3	3	0	#7
N5_blk		1	1	0	#8
N6_blk		2	2	0	#9
N7_blk		3	2	0	#10
